app/views/logs/index.html.erb

```diff
@@ -78,12 +78,16 @@
 <% else %>
   <div class="govuk-grid-row">
     <h1 class="govuk-heading-l govuk-grid-column-three-quarters">
-      <% if log_search_form.filter_option==LogSearchForm::USERNAME_FILTER_OPTION %>
-        The username "<%= log_search_form.username %>" <strong>is not reaching the GovWifi service</strong>
+      <% if !super_admin? && log_search_form.filter_option==LogSearchForm::IP_FILTER_OPTION && !current_organisation.ip_addresses.include?("#{log_search_form.ip}") %>
```

Comment:
Should this be: `!current_organisation.ip_addresses.include?(log_search_form.ip)` (without the `#{..}`)